client/state/current-user/selectors.js

```diff
@@ -104,7 +104,7 @@ export const getCurrentUserEmail = createCurrentUserSelector( 'email' );
  * @return {?Boolean}            Whether capability name is valid
  */
 export function isValidCapability( state, siteId, capability ) {
-	const capabilities = state.currentUser.capabilities[ siteId ];
+	const capabilities = get( state, [ 'currentUser', 'capabilities', siteId ] );
```

Comment:
Why do we want to use `get` here? We know the path to `state.currentUser.capabilities` will always exist because this is the shape of the store reducer. In this case, `get` will needlessly check for truthiness of each property in the complete path.